flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/JobManagerDeploymentStatus.java

```diff
@@ -54,8 +58,9 @@
                                 operatorConfiguration.getPortCheckIntervalInSec(),
                                 TimeUnit.SECONDS);
             case MISSING:
+            case ERROR:
             default:
-                return null;
+                return UpdateControl.noUpdate();
```

Comment:
An error will trigger a status update, is `noUpdate` correct here?

Reply:
seems to work as expected